libcontainer/criurpc/criurpc.proto

```diff
@@ -111,6 +111,7 @@ message criu_opts {
 	optional bool			lazy_pages		= 48;
 	optional int32			status_fd		= 49;
 	optional bool			orphan_pts_master	= 50;
+	optional string			config_file		= 51;
```

Comment:
Hi @adrianreber
I noticed that `criurpc.proto` differs with a few more lines from the current content of [images/rpc.proto](https://github.com/checkpoint-restore/criu/blob/master/images/rpc.proto).
Would it be possible to update the whole `criurpc.proto` file in this commit?

Reply:
I have already prepared a PR to vendor in go-criu which will make sure we will always have the same `criurpc.proto` as CRIU. Once this here is merged I will open the next PR with the go-criu vendoring in.

Reply:
Awesome, thanks! I plan to open a PR for `runc restore --tcp-close` but for this we need to update the proto file in both CRIU and runc.